### DISCUSSION TITLE
GitLab support for Branch Planner

### DISCUSSION THREAD

**Discussion by wbloos** [flux-iac/tofu-controller#1188] [Ideas]
We love your Branch Planner functionality, it makes the TF-controller a feasible tool for us. We'd love to use it, but we're on GitLab.
On https://weaveworks.github.io/tf-controller/branch-planner/branch-planner-tfc-integration-getting-started/ you mention:
> For now, Branch Planner only supports GitHub as the Git provider. We plan to add other Git providers in time.

Do you have a timeline concerning GitLab support?

**Comment by chanwit** [comment]
The project has become a community-driven and is short of resources to implement new features at the moment. Please consider contributing if you'd like to see things happen.
